superset/migrations/versions/b56500de1855_add_uuid_column_to_import_mixin.py

```diff
@@ -71,20 +74,31 @@ class ImportMixin:
 
 models["dashboards"].position_json = sa.Column(utils.MediumText())
 
+default_batch_size = int(os.environ.get("BATCH_SIZE", 200))
```

Comment:
This number is not the bigger the better. I tested many different numbers (100, 200, 500, 1000, 2000) and 200 seems to be working the best (but obviously this will depend on the machine, too).